Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -26,7 +26,12 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
         sender.setTitle("Processing...", for: .disabled)
         sender.isEnabled = false
 
-        let request = BTPayPalCheckoutRequest(amount: "4.30")
+        let request = BTPayPalCheckoutRequest(amount: "5.00")
+        let lineItem = BTPayPalLineItem(quantity: "1", unitAmount: "5.00", name: "item one 1234567", kind: .debit)
+        lineItem.upcCode = "123456789"
+        lineItem.upcType = BTPayPalLineItemUpcType.UPC_A
+        lineItem.imageUrl = URL(string: "https://www.example.com/example.jpg")
```

Comment:
I would expect to get an error building the demo app here since this property changed to `imageURL`